pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -2288,6 +2288,12 @@ public class ServiceConfiguration implements PulsarConfiguration {
     private int webSocketMaxTextFrameSize = 1048576;
 
     /**** --- Metrics. --- ****/
+    @FieldContext(
+            category = CATEGORY_METRICS,
+            doc = "Whether the '/metrics' endpoint requires authentication. Defaults to false."
+                    + "'authenticationEnabled' must also be set for this to take effect."
+    )
+    private boolean authenticateMetricsEndpoint = false;
```

Comment:
@liudezhi2098  We may miss the configuration `authenticateMetricsEndpoint` in proxy.conf.